components/markdown/markdown.tsx

```diff
@@ -116,11 +122,21 @@ export default class Markdown extends React.PureComponent<Props> {
         proxyImages: true,
         imagesMetadata: {},
         postId: '', // Needed to avoid proptypes console errors for cases like channel header, which doesn't have a proper value
+        editedAt: 0,
     }
 
     render() {
+        const {postId, editedAt, message} = this.props;
         if (!this.props.enableFormatting) {
-            return <span>{this.props.message}</span>;
+            return (
+                <span>
+                    {this.props.message}
+                    <PostEditedIndicator
```

Comment:
0/5 Markdown is a component way more general. I wonder if it makes sense to handle this difference somewhere else, and not in this component. Probably not possible, but still, feels weird.